core/domain/collection_domain_test.py

```diff
@@ -473,8 +493,9 @@ def test_add_delete_swap_nodes(self):
         collection.delete_node('test_exp')
         self.assertEqual(len(collection.nodes), 0)
 
-    def test_update_collection_contents_from_model(self):
-        versioned_collection_contents = {
+    def test_update_collection_contents_from_model(self) -> None:
+        versioned_collection_contents: (
+            collection_domain.VersionedCollectionDict) = {
             'schema_version': 1,
             'collection_contents': {}
```

Comment:
before mypy assumes `collection_contents` to be object, but it's not a object. it's a  collectionVersionedDict which is defined above. So, that's why this is annotated with `collection_domain.VersionedCollectionDict`